medical-system/src/components/shared/Sidebar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from "react-router-dom";
+import { ROUTES } from "../../router/routes";
+
+import {
+  MdDashboard,
+  MdLocalHospital,
+  MdPeople,
+  MdSettings,
+} from "react-icons/md";
+import { FaRegCalendarPlus } from "react-icons/fa";
+import { CgProfile } from "react-icons/cg";
+import { FaUserDoctor } from "react-icons/fa6";
+import { IconType } from "react-icons";
+
+export function Sidebar() {
+  const SIDEBAR_LINKS: {
+    id: number;
+    path: string;
+    name: string;
+    icon?: IconType;
+  }[] = [
+    { id: 1, path: ROUTES.dashboard, name: "Dashboard", icon: MdDashboard },
+    { id: 2, path: ROUTES.patients, name: "Patients", icon: MdPeople },
+    { id: 3, path: ROUTES.doctors, name: "Doctors", icon: FaUserDoctor },
+    {
+      id: 4,
+      path: ROUTES.laboratorist,
+      name: "Laboratorist",
+      icon: MdLocalHospital,
+    },
+    { id: 5, path: ROUTES.calendar, name: "Calendar", icon: FaRegCalendarPlus },
+    { id: 6, path: ROUTES.profile, name: "Profile", icon: CgProfile },
+    { id: 7, path: ROUTES.settings, name: "Settings", icon: MdSettings },
+  ];
```

Comment:
To można by było utrzymywać w jakims osobnym pliku, np w folderze constants -> sidebarLinks.ts, nie potrzebujemy w nim żadnych elementów Reactowych wiec moze to byc spokojnie poza komponentem.

Dodatkowo, w jakim celu ustalane jest tutaj ID?